tensorflow/core/api_def/base_api/api_def_Exp.pbtxt

```diff
@@ -1,4 +1,30 @@
 op {
   graph_op_name: "Exp"
   summary: "Computes exponential of x element-wise.  \\\\(y = e^x\\\\)."
+  description: <<END
+  This function computes exponential of every element in the input tensor.
+  i.e. exp(x) or e^(x), where `x` is the input tensor.
+  `e` denotes Euler's number and is approximately equal to 2.718281.
+  Output is positive for any real input.
+
+  ```python
+  x = tf.constant(2.0)
+  tf.math.exp(x) ==> 7.389056
+
+  x = tf.constant([2.0, 8.0])
+  tf.math.exp(x) ==> array([7.389056, 2980.958], dtype=float32)
+  ```
+
+  For complex numbers, say 1+1j, real part is calculated like above ( e^(1) = 2.7182818284590).
+  Imaginary part e^(1j) will be calculated as, cos 1 + i sin 1 (e^ai = cos a + i sin a) which is
+  0.54030230586+0.8414709848j.
+
```

Comment:
Please put all equations in backticks to make them render as code. This is very hard to read without that sort of font-based highlighting.

The math in this example is wrong. The example text is confusing and misleading, it needs to be rewritten.

This is the correct trigonometric replacement:  e^(x+iy) = e^x * e^iy = e^x * (cos y + i sin y).